main/main.js

```diff
@@ -367,14 +382,14 @@ app.on('continue-activity', function(e, type, userInfo, details) {
   }
 })
 
-app.on('second-instance', function (e, argv, workingDir) {
+app.on('second-instance', function (e, argv, workingDir, additionalData) {
```

Comment:
`argv` here should be the arguments of the second instance already - why do they need to be passed in with `additionalData`?